reminder/module.py

```diff
@@ -0,0 +1,465 @@
+        reminders = []
+
+        for item in query:
+            author = await self._get_member(item.guild_id, item.author_id)
+            remind = await self._get_member(item.guild_id, item.remind_id)
+
+            author_name = author.display_name if author is not None else "(unknown)"
+            remind_name = remind.display_name if remind is not None else "(unknown)"
+
+            reminder = ReminderDummy()
+            reminder.idx = item.idx
+            reminder.author_name = author_name
+            reminder.remind_name = remind_name
+            reminder.remind_date = item.remind_date
+            reminder.status = item.status.name
+            reminder.url = item.permalink.replace("https://discord.com/channels/", "")
+
+            reminders.append(reminder)
+
+        table_pages: List[str] = utils.text.create_table(
+            reminders,
+            {
+                "idx": _(ctx, "Reminder ID"),
+                "author_name": _(ctx, "Author"),
+                "remind_name": _(ctx, "Reminded"),
+                "remind_date": _(ctx, "Remind date"),
+                "status": _(ctx, "Status"),
+                "url": "https://discord.com/channels/",
+            },
+        )
+
+        for table_page in table_pages:
+            await ctx.send("```" + table_page + "```")
+
+    async def _remind(self, item: ReminderItem):
+        reminded_user = await self._get_member(item.guild_id, item.remind_id)
+
+        if reminded_user is None:
+            item.status = ReminderStatus.FAILED
+            item.save()
+            await bot_log.warning(
+                item.remind_id,
+                item.guild_id,
+                "Unable to remind user - member out of bot reach.",
+            )
+            return
+
+        tc = i18n.TranslationContext(item.guild_id, item.remind_id)
+
+        embed = await self._get_embed(utx, item)
+
+        try:
+            await reminded_user.send(embed=embed)
+        except (HTTPException, Forbidden):
+            item.status = ReminderStatus.FAILED
+            item.save()
+            await bot_log.warning(
+                item.remind_id,
+                item.guild_id,
+                "Unable to remind user - blocked PM or not enough permissions.",
+            )
+
+        item.status = ReminderStatus.REMINDED
+        item.save()
+
+        await guild_log.info(
+            reminded_user,
+            self.bot.get_guild(item.guild_id),
+            "Reminder ID {id} succesfully sent to {user}".format(
+                id=item.idx, user=reminded_user.display_name
+            ),
+        )
+
+    async def _get_member(self, guild_id: int, user_id: int):
+        guild = self.bot.get_guild(guild_id)
+        user = guild.get_member(user_id)
+
+        if user is None:
+            try:
+                user = await self.bot.fetch_user(user_id)
+            except nextcord.errors.NotFound:
+                pass
+
+        return user
+
+
+class ReminderDummy:
+    pass
```

Comment:
what is this

Reply:
Used in `_send_reminder_list` to create object with atributes, 'cause `utils.text.create_table` does not support dictionary, only objects as items.

Reply:
well the query items already are objects so we should figure out how to use those instead of going around it like this

Reply:
We would have to get data from API, such as users and their names. It would be easier if @sinus-x would add dictionary support to create_table.

Reply:
Create a ticket for that, please.

Reply:
Done.